src/editor/tools/action_args.h

```diff
@@ -53,24 +53,18 @@ struct EditorActionArgs {
 	Widelands::Extent new_map_size;                        // resize tool
 
 	struct ResourceState {
-		Widelands::FCoords location;
+		Widelands::Coords location;
```

Comment:
No need for floating point here?

Reply:
The F in FCoords means that the `Coords` has a `Field*` attached to it. However when resizing, the memory address of fields may change so this pointer would be invalidated. The `Field*` is now instead fetched afresh when needed.